index.bs

```diff
@@ -396,3 +422,81 @@ Acknowledgements {#ack}
 
 Cure53's [[DOMPURIFY]] is a clear inspiration for the API this document
 describes, as is Internet Explorer's {{window.toStaticHTML()}}.
+
+
+
+Appendix A: Interactions between Sanitizer And Trusted Types {#tt}
+======================================================
+
+<em>This section is not normative.</em>
+
+[[TrustedTypes]] can interact in helpful ways with the Sanitizer API. How to
+best do so will likely depend on the intended use of both. Because the
+Sanitizer API is meant to remove all scripting content, the default is to not
+consider the Sanitizer Trusted Types sensitive.
+
+The predominant use case would be to sanitize strings and use the resulting
+fragments directly, just as without Trusted Types:
+
+```js
+const untrusted_input = "....";
+some_node.AppendChildren(sanitizer.santize(untrusted_input));
+```
+
+For cases where handling of a string is desired, a sanitizing policy can easily
+be created using the `bound` method.
+
+```js
+const my_config = { ... };
+const sanitizing_policy = trustedTypes.createPolicy("apolicyname", {
+  createHTML: new Sanitizer(my_config).bound() });
+
+const untrusted_input = "....";
+some_node.innerHTML = sanitizing_policy.createHTML(untrusted_input);
+```
+
+If a sanitizing policy is installed as the default policy, this would
+auto-sanitize any HTML assignments.
+
+```js
+trustedTypes.createPolicy("default", { createHTML: new Sanitizer().bound() });
+
+const untrusted_input = "<em>Bobby <script>console.log(1);</script> Tables</em>";
+some_node.innerHTML = untrusted_input;  // Assigns "<em>Bobby  Tables</em>".
+```
+
+Note: Using a default policy - since it applies to the entire document and
+all dependencies - needs to be carefully considered. It is oftentimes best
+used as a transition tool, used only until an application has been made fully
+Trusted Types compliant.
+
+However, it may not always be in the developers interest to allow direct
+creation of HTML from any Sanitizer instance, especially if the Trusted Type
+policies intended for use with the document are meant to block content beyond
+direct script applications. To accomodate this case, Trusted Types' CSP
```

Comment:
I'm not sure this would have the effect we're seeking for. Applications that want to put stringer-than-XSS restrictions on their DOM, already have an issue with regular DOM APIs. Guarding the sanitizer _input_ with TT doesn't buy them much, as the problem they face is different - how to make sure that *all* access to the DOM is limited. This pattern would suggest they go through `TT(custompolicy)->Sanitizer(customconfig)->DOM`, but the the easier access pattern, and probably a source of vulns is to do `strings->DOM`. 

The challenge in enforcing such restrictions is to make sure you never *skip* calling a sanitizer, and the input type parameter doesn't give you that. 

There's also a secondary concern that `TrustedHTML` already communicates that your data is ready for DOM, even more than strings, and that might give a suggestion not to sanitize.

I think the sanitizer always accept a string on input, the data is always suspicious, that's what the sanitizer is for.